web/source/osk/visualKeyboard.ts

```diff
@@ -2365,7 +2365,8 @@ namespace com.keyman.osk {
       canvas.height = 2.3 * xHeight;
 
       kts.top = 'auto';
-      kts.bottom = (y0 + h0 - xTop - xHeight)+'px';
+      // Matches how the subkey positioning is set.
+      kts.bottom = (parseInt(key.style.bottom, 10))+'px';
```

Comment:
Compare against the `showSubKeys` version:

https://github.com/keymanapp/keyman/blob/4da1454937c498c9cf59a55afbabd14bffa89dba/web/source/osk/visualKeyboard.ts#L1509-L1512

The key preview's 'bottom' needs to match the bottom of the base key, not the top - this is why the "height" part of the line is not included here.